src/versioned_util.ml

```diff
@@ -9,11 +9,10 @@ let mk_loc ~loc txt = {Location.loc; txt}
 
 let map_loc ~f {Location.loc; txt} = {Location.loc; txt= f txt}
 
-let check_modname ~loc name =
-  if name = "Stable" then name
-  else
-    Location.raise_errorf ~loc
-      "Expected a module named Stable, but got a module named %s." name
+let check_modname ~loc name:string =
+  match name with
+  | "Stable" -> name
+  | _ -> Location.raise_errorf ~loc "Expected a module named Stable, but got a module named %s." name
```

Comment:
Can you reformat this?

Reply:
I figured out that I couldn't format things without an `.ocamlformat` file at the root, so I added one in the PR